tests/functional/cephx_keyrotation/test_cephx_keyrotation_negative.py

```
@@ -0,0 +1,816 @@
+                restored = True
+            if self._deleted_osd_auth_entity:
+                entity = self._deleted_osd_auth_entity
+                log.info(
+                    f"Teardown: ensuring deleted OSD auth entity {entity} is restored"
+                )
+                try:
+                    if rotator.ensure_osd_auth_entity_restored(entity):
+                        rotator.trigger_cephcluster_reconcile()
+                        need_full_recovery = True
+                    restored = True
+                except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)